### PR TITLE
Add demographic validation to new dependent form

### PR DESCRIPTION
**Pull request by jacobkagon**
### Master Pivotal ticket
* https://www.pivotaltracker.com/n/projects/2502930/stories/179998170

### Child Redmine ticket(s)
* ()
* ()

### Local build result

```
bundle exec rake parallel:spec[4]
bundle exec cucumber
```

### Latest rebase/merge tag
*

---

### Data ticket(s) Followup
* (redmine links here - optional)

### Related Pull Requests
* (github links here - optional)

---

#### Peer Review
* (For code review)

#### Functional Testing
* (For testing locally)

#### Deployment
* (for release manager and/or build manager)

### CODE REVIEW

File: app/assets/javascripts/demographics_fields.js [hunk 1]
```diff
@@ -423,10 +423,8 @@ var PersonValidations = (function (window, undefined) {
   }
 
   function validationForVlpDocuments(e) {
-    if (
-      $('#immigration_doc_type').val() == '' &&
-      validation_for_person_or_dependent()
-    ) {
+    console.log(validation_for_person_or_dependent());
```
Comment (markgoho) [inline app/assets/javascripts/demographics_fields.js:426]: remove console.log

**Review by markgoho — CHANGES_REQUESTED** [review]

File: app/assets/javascripts/demographics_fields.js (hunk 1 above)
Comment (jacobkagon) [reply app/assets/javascripts/demographics_fields.js:426]: Will do

**Review by bill-transue — DISMISSED** [review]
Fixes dependent VLP JS. Good work @jacobkagon and good spot on the `console.log` @markgoho!

**Review by bill-transue — APPROVED** [review]
Fixes dependent VLP JS. Good work @jacobkagon!

**Review by markgoho — APPROVED** [review]

**Pull request merged**